src/WebJobs.Script.Abstractions/WebJobs.Script.Abstractions.csproj

```diff
@@ -2,6 +2,7 @@
   <Import Project="..\..\build\common.props" />
     <PropertyGroup>
     <Version>1.0.2-preview</Version>
+    <BuildNumber Condition=" '$(BuildNumber)' == '' ">1</BuildNumber>
```

Comment:
Why are we initializing build numbers with 1 here?

Reply:
Only if `$(BuildNumber)` is not set from the yml.

Reply:
Resolving since we clarified this during conversation.